src/utils/get-default-branch.ts

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2021 Expedia, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { octokit } from '../octokit';
+import { context } from '@actions/github';
+
+export const getDefaultBranch = async () => {
+  const {
+    data: { default_branch }
+  } = await octokit.repos.get({ ...context.repo });
+  return default_branch;
+};
```

Comment:
This was just a quick refactor--extracted a method into a util file that I thought should be a shared util